bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveIndicatorCommands.java

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Honestly don't know. Being a newbie at this all I did was to copy what was implemented in the BasicClass and adapt it to the indicator class.